src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -308,9 +318,17 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
 
                     # drop the metadata columns and generate the 99.9% quantile values for the FOV
                     fov_full_pixel_data = pixel_mat_data.drop(columns=cols_to_drop)
-                    quant_dat[fov] = fov_full_pixel_data.replace(
-                        0, np.nan
-                    ).quantile(q=0.999, axis=0)
+                    quant_dat_fov = fov_full_pixel_data.replace(0, np.nan).quantile(
+                        q=channel_percentile_postnorm, axis=0).rename(fov)
+                    quant_dat_fov.index.name = "channel"
+
+                    # read in previously processed fov quantile values or initialize new df
+                    quant_dat_all = pd.read_csv(quantile_path, index_col="channel") \
+                        if os.path.exists(quantile_path) else pd.DataFrame()
+                    # update the file with the newly processed fov quantile value
+                    quant_dat_all = quant_dat_all.merge(quant_dat_fov, how="outer",
+                                                        left_index=True, right_index=True)
+                    quant_dat_all.to_csv(quantile_path)
```

Comment:
Make sure to set `index=False` to prevent an additional indexing column from appearing.

Reply:
I need the index to set to true since that's where the channel names are stored. Rather than included it as a column which would just later need to be removed when taking the column averages.